kie-aries-blueprint/pom.xml

```diff
@@ -33,6 +33,12 @@
       <artifactId>org.apache.aries.blueprint.api</artifactId>
       <scope>compile</scope>
     </dependency>
+    <!--<dependency>-->
```

Comment:
Agreed, I'll remove this. It is just a leftover when I was experimenting with different solutions.